packages/botonic-react/src/components/message.jsx

```diff
@@ -94,10 +94,26 @@ export const Message = props => {
     id: props.id || uuidv4(),
   })
 
-  const replies = React.Children.toArray(children).filter(e => e.type === Reply)
-  const buttons = React.Children.toArray(children).filter(
-    e => e.type === Button
-  )
+  const withParentId = (element, parentId) => {
+    return {
+      ...element,
+      ...{
+        props: {
+          ...element.props,
+          ...{ parentId: parentId },
+        },
```

Comment:
As @asastre said before, you don't need to `...{a: b}`, you can just set these key:values directly